File: Desarrollo_de_apps/Practica1/C#/DoubleLinkedList/DoubleLinkedList/Node.cs
using System;
namespace DoubleLinkedList
{
    public class Node
    {
        //Atributos de clase
        public string value;
        public Node next;
        public Node before;

        //Constructor de la clase
        public Node(string newValue)
        {
            value = newValue;
            next = null;
            before = null;
        }

        //Método para la comparación de los valores de los nodos
        public int CompareTo(Node other)
        {
            return string.Compare(value, other.value, StringComparison.Ordinal);
        }
    }
}

File: Desarrollo_de_apps/Practica1/JAVA/DoubleLinkedList/src/doublelinkedlist/Node.java

package doublelinkedlist;

public class Node {
    //Atributos de clase
    String value;
    Node next;
    Node before;
    
    //Constructor de la clase
    public Node(String newValue){
        value = newValue;
        next = null;
        before = null;
    }
    
    //Mètodo para comparar el valor de los nodos
    public int compareTo(Node other){
        return value.compareTo(other.value);
    }
}
